### ISSUE TITLE
Provide more contextual information for "artwork" on the p5 homepage

### ISSUE THREAD

**Issue by iamjessklein** [processing/p5.js-website#232]
On the homepage right now there are these beautiful p5 art pieces that are interesting and explanatory. They provide a show vs. tell approach to explaining how useful and powerful p5 could be in the hands of talented artist. 

I was thinking that it would be great to add more context to this. For one, providing a link to even just a blogpost from the "tombstone" information on the bottom left corner of the page. 

![artistname](https://cloud.githubusercontent.com/assets/535012/13886509/9648ee0a-ed0d-11e5-9c38-110c9e3c233d.png)

Currently it links to this (the artist's portfolio page):

<img width="1035" alt="screen shot 2016-03-18 at 1 18 45 pm" src="https://cloud.githubusercontent.com/assets/535012/13886531/b06f0a62-ed0d-11e5-9084-b1f420de4f94.png">

I mentioned this at the p5 hangout today and @lmccart and @shiffman and I riffed a bit on how neat it would be to encourage contributors to in share their progress on their projects in a "how to"/ tutorial format - a la [make magazine](http://makezine.com/projects/howtoons-marshmallow-cannon/). 

Basically, this is a great opportunity space for community sourced tutorials, demos and in fact a possibly more integrated connection to the [reference](http://p5js.org/reference/) section of the site. This could also be a great way to populate our "examples".
